pkg/runutil/runutil_test.go

```diff
@@ -117,7 +117,7 @@ func TestCloseMoreThanOnce(t *testing.T) {
 
 	CloseWithLogOnErr(lc, r, "should not be called")
 	CloseWithLogOnErr(lc, r, "should not be called")
-	testutil.Equals(t, true, lc.WasCalled)
+	testutil.Equals(t, false, lc.WasCalled)
```

Comment:
The indirect update of github.com/pkg/errors from v0.8.1 to v0.9.1 fixed a dormant issue: prior to this update, `CloseWithLogOnErr()` wouldn't catch wrapped `os.ErrClosed` as expected here

https://github.com/openshift/telemeter/blob/987128a17c198270ff22cf36ff7b7df396f51826/pkg/runutil/runutil.go#L49-L52